ledger/execution/regular_tx_execution_test.go

```diff
@@ -1275,6 +1275,7 @@ func TestSplitRuleHundredPercSplits(t *testing.T) {
 	initiator.Balance = types.Coins{TFuelWei: big.NewInt(10000 * txFee), ThetaWei: big.NewInt(0)}
 	et.acc2State(initiator)
 
+	// 100% of the payment are split, 0% left for the target account
```

Comment:
is percentage not 70%? I see Percentage: 70

Reply:
A few lines down, Bob splits the remaining 30%. So 100% of the payment are split between Alice and Bob, leaving nothing for Carol, the ServicePayment target account.